<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page2_login</class>
 <widget class="QMainWindow" name="page2_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>721</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>user name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Password_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Password_lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Verification_Password_lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="verificationpassword_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Verification Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signin_pushButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>400</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>sign in</string>
     </property>
    </widget>
    <widget class="QLabel" name="Question_label">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>400</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Do not have account?</string>
     </property>
    </widget>
    <widget class="QLabel" name="capcha_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="capcha_pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>capcha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="capcha_lineEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="next_pushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>330</y>
       <width>221</width>
       <height>86</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="showhide_pushButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="symbol_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>185</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>symbol</string>
     </property>
    </widget>
    <widget class="QLabel" name="character_label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>190</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>7 character</string>
     </property>
    </widget>
    <widget class="QLabel" name="verification_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>verification password</string>
     </property>
    </widget>
    <widget class="QLabel" name="check_username_label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>username has been exist</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
